pczt/src/roles/creator/mod.rs

```diff
@@ -0,0 +1,60 @@
+use crate::{Pczt, V5_TX_VERSION, V5_VERSION_GROUP_ID};
+
+pub struct Creator {
+    tx_version: u32,
+    version_group_id: u32,
+    consensus_branch_id: u32,
+    expiry_height: u32,
+    orchard_flags: u8,
+}
+
+impl Creator {
+    pub fn new(consensus_branch_id: u32, expiry_height: u32) -> Self {
+        Self {
+            // Default to v5 transaction format.
+            tx_version: V5_TX_VERSION,
+            version_group_id: V5_VERSION_GROUP_ID,
+            consensus_branch_id,
+            expiry_height,
+            // Spends and outputs enabled.
+            orchard_flags: 0b0000_0011,
```

Comment:
It could be nice to use bitflags here (see [`orchard::Flags`](https://github.com/ZcashFoundation/zebra/blob/main/zebra-chain/src/orchard/shielded_data.rs#L227) in `zebra-chain`).